game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -229,11 +235,17 @@ public void finishRoomTransition()
     @Override
     public void render(float delta)
     {
-
         game.player.pushCoordinatesToSprite();
+        for (AbstractPerson n : currentNPCS) {
+            n.pushCoordinatesToSprite();
+
+        }
 
         if (changeMap) {
             tiledMapRenderer.setMap(game.player.getRoom().getTiledMap());
+            tiledMapRenderer.clearPeople();
+            tiledMapRenderer.addPerson((List<AbstractPerson>) ((List<? extends AbstractPerson>) currentNPCS));
```

Comment:
I like that we have it working, but this line here is quite untidy... I have sorted it, but everything is now an AbstractPerson. the `currentNPCs` object is `List<AbstractPerson>` do we want that?